getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Getting started with VPP networking
+description: Install Calico with the VPP dataplane on a Kubernetes cluster
+canonical_url: '/getting-started/kubernetes/vpp/getting-started'
+---
+
+
+### Big picture
+
+Install {{site.prodname}} and enable the tech preview of the new VPP dataplane.
+
+> **Warning!** The VPP dataplane is a tech preview and should not be used in production clusters. It has had limited testing and it will contain bugs (please report these on the [Calico Users slack](https://calicousers.slack.com/archives/C017220EXU1) or [Github](https://github.com/projectcalico/vpp-dataplane/issues)).  In addition, it does not support all the features of {{site.prodname}} and it is missing some security features such as Host Endpoint policies.
+{: .alert .alert-danger }
+
+### Value
+
+The new VPP dataplane mode has several advantages over standard linux networking pipeline mode:
+
+* It scales to higher throughput, expecially with encryption (Wireguard) enabled.
+* It offers IPsec encryption for even better encryption performance.
+* It has native support for Kubernetes services (without needing kube-proxy) that:
+
+  * Reduces first packet latency for packets to services.
+  * Preserves external client source IP addresses all the way to the pod.
+
+The VPP dataplane is entirely compatible with the other Calico dataplanes, meaning that it is possible to have a cluster with some VPP-enabled nodes and some regular nodes. This makes it easy to migrate a cluster from Linux or eBPF Calico networking to VPP networking.
+
+Trying out the tech preview will give you a taste of these benefits and an opportunity to give feedback to the Calico/VPP team.
+
+In the future, the VPP dataplane will offer additional features for network-intensive applications, such as providing `memif` userspace packet interfaces to the pods (instead of regular Linux network devices).
+
+
+### Before you begin...
+
+In the tech preview release, VPP mode has the following pre-requisites:
+
+- Must use the Calico CNI plugin and Calico IPAM.  It is not yet compatible with third-party CNI plugins (AWS CNI/Azure CNI/GKE CNI/flannel etc).
+- The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **VPP dataplane**
+
+### Concepts
+
+#### VPP
+
+The Vector Packet Processor (VPP) is a high-performance, open-source  userspace network dataplane written in C and developped under the [fd.io](https://fd.io) umbrella. It supports many standard networking features (L2, L3 routing, NAT, encapsulations), and it is also easily extensible through a plugin system. To use it as a dataplane for {{site.prodname}}, we developped a couple plugins to efficiently implement K8s services load balancing and {{site.prodname}} policies.
+
+### Installing Calico-VPP on a kubernetes cluster
```

Comment:
### Install Calico-VPP on a Kubernetes cluster

If you want to try Calico-VPP and see how it works, the simplest way is to use the [VM-based test setup]({{ site.baseurl }}/getting-started/kubernetes/vpp/vm). This requires an Ubuntu 18.04 or 20.04 host, and creates a small, 3-VM cluster with virtio interfaces that you can play with.

If you are using hosted Kubernetes clusters in public clouds, we provide ready-to go configurations for the major cloud platforms:
* [Amazon EKS]({{ site.baseurl }}/getting-started/kubernetes/vpp/eks)
* Google GKE: coming soon!
* Azure AKS: coming soon!

If the above doesn't apply and you want to test on a specific cluster, then you can use the generic configuration for a baremetal cluster: [Baremetal Cluster setup]({{ site.baseurl }}/getting-started/kubernetes/vpp/baremetal)

If you have a pre-existing Calico cluster, it is also possible to [migrate it to the VPP dataplane]({{ site.baseurl }}/reference/vpp/migrating-cluster).